app/src/main/java/com/keylesspalace/tusky/util/ListStatusAccessibilityDelegate.kt

```diff
@@ -0,0 +1,135 @@
+package com.keylesspalace.tusky.util
+
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import android.view.accessibility.AccessibilityEvent
+import android.view.accessibility.AccessibilityManager
+import androidx.core.view.AccessibilityDelegateCompat
+import androidx.core.view.accessibility.AccessibilityNodeInfoCompat
+import androidx.core.view.accessibility.AccessibilityNodeInfoCompat.AccessibilityActionCompat
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerViewAccessibilityDelegate
+import com.keylesspalace.tusky.R
+import com.keylesspalace.tusky.interfaces.StatusActionListener
+import com.keylesspalace.tusky.viewdata.StatusViewData
+
+// Not using lambdas because there's boxing of int then
```

Comment:
did not know that, thx

Reply:
In this particular case. Kotlin exports it's lambda interface as generic method to Java. Generics are always boxed (that's why there are `IntStream`  and such).